md-style-guide.md

```diff
@@ -0,0 +1,130 @@
+### 🚀 Custom Markdown Writing Style Guide
+
+### **Star**
+
+#### Adds a star with a label along with it. In place of `"label"` text can be added which needs to be shown. 
+
+```markdown
+<Star label="label"/>
+```
+
+#### Image
+![image](https://github.com/DhairyaMajmudar/json-schema-website/assets/124715224/936f4561-a37e-42c3-aaad-8ac463ae5d34)
```

Comment:
I'd prefer not to have images but to actual show the markdown working.  So you'll have a `<Star/>` tag in a code block (as you do), then you'd actually have another tag not in a code block so it renders.

The idea is to show it working and show how it works.

Reply:
It will only render in the website, GitHub or other markdown renders won't be able to interpret our Custom Tags

Reply:
Gotcha thanks for letting me know :D

Reply:
I had a quick chat with @gregsdennis and we agreed that the best place for this guide is inside the pages folder to be considered as a website page instead as a regular md file in the root folder, this way we can have everything rendered nicely. We envision this page to be accesed as part of the website experience. 

We can add the link to this page in the text part of the header of the blog:
<img width="812" alt="Screenshot 2024-02-28 at 21 41 17" src="https://github.com/json-schema-org/website/assets/40007659/02e50e9b-6cf7-4c74-9ec2-bbf52049d687">

Reply:
@DhairyaMajmudar Thanks for the flexibility addressing additional requirements!